packages/react-generator/src/expressions/class-members/get-accessor.ts

```diff
@@ -9,10 +10,14 @@ export class GetAccessor extends BaseGetAccessor {
     return `${super.getter(componentContext)}()`;
   }
 
-  getDependency(options: toStringOptions): string[] {
-    return calculateMethodDependency(
+  getDependencyString(options: toStringOptions): string[] {
+    const dependencies = calculateMethodDependency(
       super.getDependency(options),
       options.members,
     );
+    return dependencies.reduce((arr: string[], dep) => (dep instanceof BaseClassMember
```

Comment:
For me the next is more clear:
```
 [
...arr, 
...(dep instanceof BaseClassMember ? dep.getDependencyString(options) : dep)
])
```

Reply:
if we will do such way, then if dep = 'str' will return 's', 't', 'r'